packages/webassemblyjs/src/interpreter/kernel/exec.js

```diff
@@ -240,7 +246,10 @@ export function executeStackFrame(
     }
 
     function newRuntimeError(msg) {
-      return new RuntimeError(msg);
+      const e = new RuntimeError(msg);
+      e._appstack = stack;
```

Comment:
I'm not a super fan of that. The Error object is unspecified and I hope that this will always work.